doc/source/basics.rst

```diff
@@ -1522,23 +1522,29 @@ then the more *general* one will be used as the result of the operation.
 object conversion
 ~~~~~~~~~~~~~~~~~
 
-:meth:`~DataFrame.convert_objects` is a method to try to force conversion of types from the ``object`` dtype to other types.
-To force conversion of specific types that are *number like*, e.g. could be a string that represents a number,
-pass ``convert_numeric=True``. This will force strings and numbers alike to be numbers if possible, otherwise
-they will be set to ``np.nan``.
+.. note::
+
+The syntax of :meth:`~DataFrame.convert_objects`  changed in 0.17.0.
```

Comment:
This line has to be intended to belong to the note.

Further, can you add a link to the whatsnew notes to have an explanation what changed